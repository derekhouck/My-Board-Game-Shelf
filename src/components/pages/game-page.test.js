import React from 'react';
import { shallow } from 'enzyme';
import GamePage from './game-page';

describe('<GamePage />', function () {
  it('renders without crashing', () => {
    const wrapper = shallow(<GamePage />);
    expect(wrapper).toMatchSnapshot();
  })
});