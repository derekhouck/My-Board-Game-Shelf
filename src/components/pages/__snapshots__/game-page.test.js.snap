// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GamePage /> renders without crashing 1`] = `ShallowWrapper {}`;
